### ISSUE TITLE
Table headers in a data table must refer to data cells

### ISSUE THREAD

**Issue by bmbc-dev-detec** [LocalGovIMS/localgov-ims#327]
Table data cells are missing or empty in some tables, such as on the Transfer and Suspense pages.

Screen readers have a specific way of announcing tables. When tables are not properly marked up, this creates the opportunity for confusing or inaccurate screen reader output. If tables are not marked up semantically and do not have the correct header structure, screen reader users cannot correctly perceive the relationships between the cells and their contents visually.
